extensions/emmet/src/util.ts

```diff
@@ -128,6 +128,97 @@ export function parseDocument(document: vscode.TextDocument, showError: boolean
 	return undefined;
 }
 
+export function parsePartialStylesheet(document: vscode.TextDocument, position: vscode.Position): Node | undefined {
+
+	let startPosition = new vscode.Position(0, 0);
+	let endPosition = new vscode.Position(document.lineCount - 1, document.lineAt(document.lineCount - 1).text.length);
+	const closeBrace = 125;
+	const openBrace = 123;
+	let slash = 47;
+	let star = 42;
+
+	// Go forward until we found a closing brace.
+	let stream = new DocumentStreamReader(document, position);
+	while (!stream.eof()) {
+		if (stream.eat(closeBrace)) {
```

Comment:
Can we combine this `if` with the `while`?
`while (!stream.eof() && !stream.eat(closeBrace))`

Reply:
Yes we can